airbyte-integrations/connectors/source-simplecast/manifest.yaml

```diff
@@ -0,0 +1,684 @@
+version: 5.7.5
+
+type: DeclarativeSource
+
+description: >-
+  Say hello to the modern end-to-end podcasting platform. Simplecast remains the
+  easiest way to get audio out to the world—with one-click publishing to Apple
+  Podcasts Apple Podcasts , Spotify Spotify, or wherever your audience
+  listens—and the best way for podcasters to cash in on their content.
+
+check:
+  type: CheckStream
+  stream_names:
+    - podcasts
+
+definitions:
+  streams:
+    podcasts:
+      type: DeclarativeStream
+      name: podcasts
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /podcasts/
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - collection
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: offset
+          pagination_strategy:
+            type: OffsetIncrement
+            inject_on_first_request: true
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/podcasts"
+    episodes:
+      type: DeclarativeStream
+      name: episodes
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /podcasts/{{ config['podcast_id'] }}/episodes
```

Comment:
Can you configure the stream `podcasts` to be parent of `episodes` and `analytics`? This way you don't limit to ingest data from a single podcast.